utils/protobuf-schema-utilities/src/main/java/io/apicurio/registry/utils/protobuf/schema/FileDescriptorUtils.java

```diff
@@ -537,6 +584,19 @@ private static EnumElement toEnum(EnumDescriptorProto ed) {
         return new EnumElement(DEFAULT_LOCATION, name, "", options.build(), constants.build());
     }
 
+    private static ServiceElement toService(DescriptorProtos.ServiceDescriptorProto sv) {
+        String name = sv.getName();
+        ImmutableList.Builder<RpcElement> rpcs = ImmutableList.builder();
+        for (MethodDescriptorProto md : sv.getMethodList()) {
+            rpcs.add(new RpcElement(DEFAULT_LOCATION, md.getName(), "", md.getInputType(),
+                    md.getOutputType(), md.getClientStreaming(), md.getServerStreaming(),
+                    getOptionList(md.getOptions().hasDeprecated(), md.getOptions().getDeprecated())));
+        }
+
+        return new ServiceElement(DEFAULT_LOCATION, name, "", rpcs.build(),
+                getOptionList(sv.getOptions().hasDeprecated(), sv.getOptions().getDeprecated()));
```

Comment:
Can there be other options?

Reply:
I checked the Service class and there's only deprecated.